static/app/components/replays/unmaskAlert.tsx

```diff
@@ -44,7 +56,7 @@ const DOMAlertContainer = styled('div')`
   pointer-events: none;
 `;
 
-const DOMAlert = styled('div')`
+const UnmaskAlert = styled('div')`
```

Comment:
ah, naming collision. internal names like this could be less verbose, `Alert` or `Layout` kind of a name.

Reply:
i just changed this to `Alert`!